### PR TITLE
Ba add footer

### PR DESCRIPTION
**Pull request by abernethyb**
Added footer to all pages.  CSS is not working on all of them and it is not appearing on the newsfeed page, so we will need to address the newsfeed page later.  Let me know what you all think.

### CODE REVIEW

File: src/index.html [hunk 1]
```diff
@@ -222,6 +222,15 @@ <h5 class="card-title">Card title</h5>
...
             </section>
             <section class="text-center section-3">
+                <div >
+                    <h3>List of Aliases</h3>
+                    <p>Name One</p>
+                    <p>Name Two</p>
+                    <p>Name Three</p>
+                </div>
+                <div>
+                    <h3>Country of Residence</h3>
```
Comment (Adanieljohnson91) [inline src/index.html]: Delete code

**Review by bdanielle092 — CHANGES_REQUESTED** [review]
good job

**Review by Adanieljohnson91 — CHANGES_REQUESTED** [review]
Notes annotated lines 225-232

**Review by bdanielle092 — APPROVED** [review]
awesome

**Review by Adanieljohnson91 — APPROVED** [review]
Looks good

**Pull request merged**